### PR TITLE
add new indexes for ConsignmentAPI

### PR DESCRIPTION
**Pull request by paulyoung84**
@TomJKing could you review? Cheers.

### CODE REVIEW

File: lambda/src/main/resources/db/migration/V119__getConsignment_further_indexes.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+-- Add index in tables File FileMetadata and FileStatus to improve getConsignment query 
+
+CREATE INDEX IF NOT EXISTS file_allfields_idx 
+ON public."File" 
+USING btree ("ConsignmentId", "FileId", "FileType","FileName","ParentId","UserId","Datetime","ChecksumMatches");
```
Comment (TomJKing) [inline lambda/src/main/resources/db/migration/V119__getConsignment_further_indexes.sql:5]: Could you make sure there is a space between each column name, just for consistency

File: lambda/src/main/resources/db/migration/V119__getConsignment_further_indexes.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+CREATE INDEX IF NOT EXISTS filemetadata_allfields_idx 
+ON public."FileMetadata" 
+USING btree ("FileId", "PropertyName","Value","MetadataId", "Datetime","UserId");
+
+CREATE INDEX IF NOT EXISTS filestatus_allfields_idx 
+ON public."FileStatus" 
+USING btree ("FileId", "StatusType","Value","FileStatusId", "CreatedDatetime");
```
Comment (TomJKing) [inline lambda/src/main/resources/db/migration/V119__getConsignment_further_indexes.sql:13]: Could you make sure there is a space between each column name, just for consistency

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one very minor pedantic comment.

**Comment by paulyoung84** [comment]
I've added a new commit with those changes, cheers

**Comment by paulyoung84** [comment]
@vimleshtna here is the ticket to review cheers

**Review by vimleshtna — APPROVED** [review]
Looks good, thanks

**Pull request merged**